packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -585,7 +590,20 @@ export class MgtPerson extends MgtTemplatedTaskComponent {
    * @memberof MgtPerson
    */
   protected renderLoading = (): TemplateResult => {
-    return this.renderTemplate('loading', null) || html``;
+    return (
+      this.renderTemplate('loading', null) ||
+      html`
+        <div class="person-root small oneline">
+          <div class="avatar-wrapper">
+            <fluent-skeleton shimmer class="shimmer icon" shape="circle"></fluent-skeleton>
+          </div>
+          <div class="details-wrapper">
```

Comment:
there should be a check to see what the view is and set the number of lines, and if there is even a details-wrapper present. Actually it probably wants to know what the AvatarSize is too

I would expect the shimmer to take up the same amount of space as the settled rendering of the person